onnx/defs/nn/defs.cc

```diff
@@ -783,6 +877,78 @@ ONNX_OPERATOR_SET_SCHEMA(
     11,
     OpSchema().FillUsing(ConvOpSchemaGenerator("a filter")));
 
+std::function<void(OpSchema&)> UnfoldToDepthOpSchemaGenerator() {
```

Comment:
again, this op should be put in the "tensor" folder instead of "nn", it belongs to the same category as "SpaceToDepth", which is used for changing a tensor's layout.